guides/common/modules/ref_ports-and-firewall-requirements.adoc

```diff
@@ -17,70 +17,50 @@ This includes the base operating system on which {SmartProxyServer} is running.
 .Clients of {SmartProxy}
 Hosts which are clients of {SmartProxies}, other than {Project}'s integrated {SmartProxy}, do not need access to {ProjectServer}.
 ifdef::satellite[]
-For more information on {Project} Topology, see {PlanningDocURL}sect-Documentation-Architecture_Guide-Capsule_Networking[{SmartProxy} Networking] in _Planning for {ProjectNameX}_.
+For more information on {Project} Topology and an illustration of port connections, see {PlanningDocURL}sect-Documentation-Architecture_Guide-Capsule_Networking[{SmartProxy} Networking] in _Planning for {ProjectNameX}_.
 endif::[]
 
 Required ports can change based on your configuration.
 
-ifdef::satellite[]
-A matrix table of ports is available in the Red{nbsp}Hat Knowledgebase solution https://access.redhat.com/solutions/5627751[Red Hat Satellite List of Network Ports].
-endif::[]
-
 The following tables indicate the destination port and the direction of network traffic:
 
-ifdef::satellite[]
-ifeval::["{mode}" == "connected"]
-.Ports for {Project} to Red Hat CDN Communication
-[cols="24%,20%,18%,38%",options="header"]
-|====
-| Port | Protocol | Service | Required For
-| 443 | TCP | HTTPS | Subscription Management Services (access.redhat.com) and connecting to the Red{nbsp}Hat CDN (cdn.redhat.com).
+.{ProjectServer} incoming traffic
+[cols="15%,15%,15%,15%,20%,20%",options="header
 |====
-endif::[]
-endif::[]
-
+| Destination Port | Protocol | Service |Source| Required For | Description
+| 53 | TCP and UDP | DNS | DNS Servers and clients | Name resolution | DNS (optional)
+| 67 | UDP | DHCP | Client | Dynamic IP | DHCP (optional)
+| 69 | UDP | TFTP | Client | TFTP Server (optional) |
+| 443 | TCP | HTTPS | {SmartProxy} | {ProjectName} API | Communication from {SmartProxy}
 ifdef::satellite[]
-ifeval::["{mode}" == "connected"]
-{ProjectServer} needs access to the Red{nbsp}Hat CDN.
-For a list of IP addresses used by the Red{nbsp}Hat CDN (cdn.redhat.com), see the Knowledgebase article https://access.redhat.com/articles/1525183[Public CIDR Lists for Red Hat] on the Red{nbsp}Hat Customer Portal.
+// ToDo Figure out how cloud.redhat.com connects
```

Comment:
There is no incoming traffic to worry about from `cloud.redhat.com`.

Reply:
This was for incoming traffic. Should there be something for outgoing?

Reply:
There are two vectors for communication back to cloud:

 1) receptor gets enabled on the Satellite and initiates connection to the cloud and establishes a bi-directional channel
 2) the rh_cloud plugin makes API calls back up to the cloud

Reply:
Is this port 443?

Reply:
| 443 |  TCP | HTTPS | cloud.redhat.com | Red{nbsp}Hat Cloud plugin API calls ?

Reply:
Sounds good